_pages/en/contact-us.md

```diff
@@ -6,4 +6,15 @@ lang: en
 permalink: /contact-us.html
 ---
 
-{{ site.data.i18n.general.notices.feedback[page.lang]}}
+
+<!-- {{ site.data.i18n.general.notices.feedback[page.lang]}} -->
+<br/>
+All work found on this site are ideas and proposals from the IT Strategy team within ESDC. Content here does not represent the official stance of ESDC nor the Government of Canada at large.
```

Comment:
Don't need this line, it is in the yellow footer.

Reply:
it will be removed. Thanks